lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -422,35 +429,35 @@
           - Note that C(npipe) is only supported by Docker for Windows.
         type: str
         choices:
-          - 'bind'
-          - 'volume'
-          - 'tmpfs'
-          - 'npipe'
+          - bind
+          - volume
+          - tmpfs
+          - npipe
         default: volume
       read_only:
         description:
-          - 'Whether the mount should be read-only.'
+          - Whether the mount should be read-only.
         type: bool
       consistency:
         description:
-          - 'The consistency requirement for the mount.'
+          - The consistency requirement for the mount.
         type: str
         choices:
-          - 'default'
-          - 'consistent'
-          - 'cached'
-          - 'delegated'
+          - default
+          - consistent
+          - cached
+          - delegated
       propagation:
         description:
           - Propagation mode. Only valid for the C(bind) type.
         type: str
         choices:
-          - 'private'
-          - 'rprivate'
-          - 'shared'
-          - 'rshared'
-          - 'slave'
-          - 'rslave'
+          - private
+          - rprivate
```

Comment:
the same as above

Reply:
Here I would keep this order, since here the `r` variant is always next to the non-`r` variant; and the non-`r` variants are ordered alphabetically.

Reply:
ok, np, it's just a recommendation AFAIK